### PR TITLE
Supported OS+arch for orcharhino

### PR DESCRIPTION
**Pull request by maximiliankolb**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8)

### CODE REVIEW

File: guides/common/modules/ref_supported-clients-in-registration.adoc [hunk 1]
```diff
@@ -43,24 +54,12 @@ ifdef::satellite[]
...
-|{EL} |x86_64
-|{SLES} 12 |amd64
-|{SLES} 15 |aarch64, ppc64le, amd64
-|Ubuntu 16.04 |amd64
-|Ubuntu 18.04 |amd64
-|Ubuntu 20.04 |aarch64, amd64
-|Ubuntu 22.04 |aarch64, amd64
-|===
+* x86_64/amd64
+* aarch64 and ppc64le for select operating systems
```
Comment (Lennonka) [inline guides/common/modules/ref_supported-clients-in-registration.adoc:59]:
```suggestion
* aarch64 and ppc64le for selected operating systems
```
?

**Review by Lennonka — APPROVED** [review]
Mostly LGTM

File: guides/common/modules/ref_supported-clients-in-registration.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/ref_supported-clients-in-registration.adoc:59]: I think "select" is correct here: https://www.merriam-webster.com/dictionary/select

Maybe @bangelic can weigh in?

Comment (apinnick) [reply guides/common/modules/ref_supported-clients-in-registration.adoc:59]: It depends what you are trying to say. "Select" is an ambiguous word. OTOH, "Selected" implies that the user has selected these operating systems.

Maybe this:

- x86_64/amd64 is supported by all operating systems.
- aarch64 and ppc64le is supported by specific operating systems.

Comment (maximiliankolb) [reply guides/common/modules/ref_supported-clients-in-registration.adoc]: Reworded.

**Comment by Lennonka** [comment]
Cherry-picked:
- 3c0dfdc2d4..3e844df433  3.11 -> 3.11
- fb836e5909..1735fd515d  3.10 -> 3.10
- 458968af3e..320f97bf0a  3.9 -> 3.9

**Pull request merged**